### PR TITLE
Logic to compile co-presence worker script

### PR DESCRIPTION
**Pull request by ritikramuka**
added logic to compile co-presence worker script in webpack config and gulp file

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -122,4 +122,58 @@ const webConfig = {
...
+/** @type fluent container scripts web worker config */
+const webWorkerConfig = {
+    mode: "none",
+    target: "webworker", // web extensions run in a webworker context
+    entry: {
+        main: "./src/web/client/common/worker/webworker.js",
+    },
+    output: {
+        filename: "[name].js",
+        path: path.join(__dirname, "./dist/web"),
```
Comment (ritikramuka) [inline webpack.config.js:134]: @amitjoshi438 this is the location where you suggested adding the webworker compiled file in the dist folder?

File: gulpfile.mjs [hunk 2]
```diff
@@ -81,6 +81,14 @@ function compileWeb() {
         .pipe(gulp.dest(path.resolve(`${distdir}/web`)));
 }
 
+// compile the web worker for vscode for web
+function compileWorker() {
+    return gulp
+        .src(["src/web/**/*.ts"])
```
Comment (tyaginidhi) [inline gulpfile.mjs:87]: the worker path is now in common folder correct?

File: webpack.config.js (hunk 1 above)
Comment (tyaginidhi) [reply webpack.config.js:134]: better keep it in dist/common ?

Comment (tyaginidhi) [reply webpack.config.js:134]: let it be in web for now

**Review by tehcrashxor — APPROVED** [review]

**Pull request merged**